docs/tutorials/python/types_of_data_augmentation.md

```diff
@@ -335,7 +332,7 @@ And lastly, you can use [`mxnet.image.RandomOrderAug`](https://mxnet.incubator.a
 ```python
 example_image_copy = example_image.copy()
 aug_list = [
-    mx.image.RandomCropAug(size=(50, 50)),
+    mx.image.RandomCropAug(size=(250, 250)),
```

Comment:
Why change? And do we need to update image?

Reply:
the generated image was really random, this one always shows at least a part of the giraffe, which makes more sense IMO. The image can remain since it is a sensible one.